worlds/smw/Client.py

```diff
@@ -169,7 +169,75 @@ async def handle_message_queue(self, ctx):
 
         await snes_flush_writes(ctx)
 
-        return
+
+    def add_trap_to_queue(self, trap_item, trap_msg):
+        if not hasattr(self, "trap_queue"):
+            self.trap_queue = []
```

Comment:
```suggestion
        self.trap_queue = getattr(self, "trap_queue", [])
```
Mostly style difference, suggesting in case this getattr functionality was not known.